bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -166,4 +177,19 @@
 		<description>Indicates the exported kWh Consumption.</description>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="batteryChargingFromGrid">
+		<item-type>Switch</item-type>
+		<label>Start/Stop Active Battery Charging from Grid</label>
```

Comment:
Labels should be < 23 chars and 2-3 words proper capitalized.
```suggestion
		<label>Charging from Grid</label>
```

Reply:
ok